src/compute/src/memory/config.rs

```diff
@@ -157,14 +157,14 @@ mod tests {
     #[test]
     fn test_reserve_memory_bytes() {
         // at least 512 MB
```

Comment:
Should be 1GB now.

Reply:
Any reference?

Reply:
Oh, sorry. 512MB Is correct. I misread the corresponding codes.